do/loadbalancers.go

```diff
@@ -17,10 +17,12 @@ limitations under the License.
 package do
 
 import (
+	goctx "context"
```

Comment:
Is this a common abbreviation in kube land? Unless it's not, I suggest to use `context` and not to rename it.

Reply:
That alias is there to distinguish go context from godo context.

https://github.com/StackPointCloud/digitalocean-cloud-controller-manager/blob/f56f1ba771fea00550b1653680dcf77f10398d5d/do/loadbalancers.go#L20-L31

(Yes, probably the alias is unfortunate)

Reply:
Maybe we should rename that name to `doctx` (because that's how named it internally as well). Thoughts @andrewsykim ?

Reply:
sounds good. Waiting for Andrew to remove `goctx` alias and set `doctx` for `github.com/digitalocean/godo/context`

Reply:
yeah, doctx seems to make more sense as opposed to aliasing stdlib, @odacremolbap feel free to leave `goctx` for now and I'll address that in another PR :)

Reply:
ok